### ISSUE TITLE
prop.rel is broken

### ISSUE THREAD

**Issue by olivercoad** [Zaid-Ajaj/Feliz#258]
I want to add a canonical link element, but `canonical` is not yet a member of the `rel` type so I used the function instead.

```fsharp
Html.link [ prop.rel "canonical"; prop.href canonicalUrl ]
```

This results in the following error

```
Uncaught TypeError: tag[lowerCaseAttributeKey].toLowerCase is not a function
    at eval (VM113533 Helmet.js:252)
    at Array.filter (<anonymous>)
    at eval (VM113533 Helmet.js:244)
    at Array.reduce (<anonymous>)
    at getTagsFromPropsList (VM113533 Helmet.js:241)
    at reducePropsToState (VM113533 Helmet.js:317)
    at emitChange (VM113534 index.js:56)
    at SideEffect.UNSAFE_componentWillMount (VM113534 index.js:97)
    at callComponentWillMount (VM113458 react-dom.development.js:12953)
    at mountClassInstance (VM113458 react-dom.development.js:13048)
```

Workaround is to use a custom prop
```fsharp
prop.custom ("rel", "canonical")
```

**Comment by Shmew** [comment]
Oh yep, I see the issue. I'll get this pushed out asap.

**Comment by Zaid-Ajaj** [comment]
Fixed as of v1.13.2 :rocket:

**Comment by olivercoad** [comment]
Thanks for the quick fix!
